### PR TITLE
Feature/yolov10 aliases and tests

### PR DESCRIPTION
**Pull request by NickHerrig**
# Description

Please include a summary of the change and which issue is fixed or implemented. Please also include relevant motivation and context (e.g. links, docs, tickets etc.).

## Type of change

Please delete options that are not relevant.

-   [X] New feature (non-breaking change which adds functionality)
-   [X] This change requires a documentation update

## How has this change been tested, please provide a testcase or example of how you tested the change?

YOUR_ANSWER

## Any specific deployment considerations

For example, documentation changes, usability, usage/costs, secrets, etc.

## Docs

-   [ ] Docs updated? What were the changes:

### CODE REVIEW

**Review by grzegorz-roboflow — APPROVED** [review]

File: tests/inference/models_predictions_tests/conftest.py [hunk 1]
```diff
@@ -146,6 +146,17 @@ def yolonas_det_model() -> Generator[str, None, None]:
     shutil.rmtree(model_cache_dir)
 
 
+@pytest.fixture(scope="function")
+def yolov10_det_model() -> Generator[str, None, None]:
+    model_id = "yolov10_det/1"
+    model_cache_dir = fetch_and_place_model_in_cache(
+        model_id=model_id,
+        model_package_url="https://storage.googleapis.com/roboflow-tests-assets/yolov10_det.zip",
```
Comment (NickHerrig) [inline tests/inference/models_predictions_tests/conftest.py:154]: Wondering if there is another step here other than uploading this zip to google cloud.

Comment (PawelPeczek-Roboflow) [reply tests/inference/models_predictions_tests/conftest.py:154]: there should be adjustment of access policy required

Comment (PawelPeczek-Roboflow) [reply tests/inference/models_predictions_tests/conftest.py:154]: ok, was wrong - problem was rather in content of ZIP

**Review by PawelPeczek-Roboflow — APPROVED** [review]

**Review by PawelPeczek-Roboflow — CHANGES_REQUESTED** [review]
Fixed the problem with package, but confidence seems to be different

**Comment by NickHerrig** [comment]
Hey @PawelPeczek-Roboflow this should be good to go now! Thanks for helping with the zip issue.

**Review by grzegorz-roboflow — DISMISSED** [review]

**Review by PawelPeczek-Roboflow — APPROVED** [review]

**Comment by PawelPeczek-Roboflow** [comment]
syntax error I guess

**Review by PawelPeczek-Roboflow — APPROVED** [review]

**Review by grzegorz-roboflow — APPROVED** [review]

**Pull request merged**